lib/core/test/bench/db/Main.hs

```diff
@@ -157,22 +181,28 @@ import System.IO.Unsafe
 import System.Random
     ( mkStdGen, randoms )
 
+import qualified Cardano.BM.Configuration.Model as CM
+import qualified Cardano.BM.Data.BackendKind as CM
+import qualified Cardano.Wallet.Primitive.AddressDerivation.Byron as Byron
+import qualified Data.ByteArray as BA
 import qualified Data.ByteString as BS
 import qualified Data.ByteString.Char8 as B8
 import qualified Data.Map.Strict as Map
 
 main :: IO ()
-main = withUtf8Encoding $ do
+main = withUtf8Encoding $ withLogging $ \trace -> do
+    let tr = trMessageText $ filterTraceSeverity Error trace
```

Comment:
Could we use `Cardano.Wallet.Shelley.setupTracers` to get a tracer? That would be more real.

Reply:
Then we would have to relocate the benchmark code. In particular, the benchmarks are also still using the `JormungandrKey` implementations and not the `Shelley` ones for addresses.  

I originally used stuff from cardano-cli, but it was a sort of circular dependency since cardano-cli depends on cardano-core and the benchmark is part of cardano-core. It was quite annoying to compile because on any change, instead of having to simply recompile the benchmark code, it had to re-compile the entire cardano-wallet-core :|